dock-schemas.rst

```diff
@@ -0,0 +1,148 @@
+Available schemas
+==================================
+
+Currently, the following schemas are available to use in the Dock protocol:
+
+- basicUserProfile
+- contactInfo
+- contacts
+- education
+- email
+- empty
+- jobExperience
+- jobRecommendations
+- skills
+- userProfile
+
+A more detailed description of each schema can be found below.
```

Comment:
Do we need this descriptions? They don't provide any useful info other than links to schemas. Fields one can see in the schema itself  (with way more info than in this list)

Reply:
Issue with having them here is that now we have to support two different places where we define schemas and fields, while the purpose of schemas (and schemas repo) is to be self-explanatory.

Reply:
High level list is useful for biz & product, I agree that it will be a maintenance overhead for us though. I would prefer to still keep it but if you think it's easy to miss keeping this is sync then lets delete it. Its more important to have a 'correct' source that everyone refers.